packages/formation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/formation",
-  "version": "5.6.1",
+  "version": "5.7.0",
```

Comment:
formation requires scss variable overrides

Reply:
I think this needs to go up a major version. You have to update vets-website in order to build styles using these changes.